contributors.yml

```diff
@@ -21,8 +21,8 @@
 - Alarid
 - alcpereira
 - AlemTuzlak
-- alex-ketch
 - alexanderson1993
+- alex-ketch
```

Comment:
Would you mind removing these sorting changes unrelated to adding your name?  Our CI handles automatically sorting this file 👍